jarviscli/plugins/beer.py

```diff
@@ -0,0 +1,30 @@
+import os
+import requests
+from colorama import Fore
+
+# Generate a random beer for the user 
+@require(network=True)
+@plugin('beer')
+def beer(self):
+    # Introduction Messages
+    jarvis.say('')
+    jarvis.say('Hello! Jarvis here.', Fore.GREEN)
+    jarvis.say('Jarvis will provide a random beer', Fore.GREEN)
+
+    # Pull from a random beer API
+    rx = requests.get('https://api.punkapi.com/v2/beers/random')
+    x = 0
+
+    # Gross JSON fixing, needs to be refactored
+    for i in rx:
+        f = f + str(i)
+        x = x + 1
+        if x == 2
```

Comment:
This is definitly a SyntaxError and I don't really understand what you are trying to do here...